UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -955,4 +956,17 @@ public void CmdFilledDynamicInput(GameObject forGameObject, string input)
 			dynamicInput.OnInputFilled(input, playerScript);
 		}
 	}
+
+	[Command]
+	public void CmdTriggerStorageTrap(IEnumerable<ItemSlot> slots, GameObject player)
+	{
+		foreach (var slot in slots)
+		{
+			if(slot.IsEmpty) continue;
+			if (slot.ItemObject.TryGetComponent<MouseTrap>(out var trap))
+			{
+				if(trap.IsArmed) trap.TriggerTrapFromContainer(player.GetComponent<LivingHealthMasterBase>());
+			}
+		}
+	}
```

Comment:
```suggestion
	[Command]
	public void CmdTriggerStorageTrap(GameObject backPack)
	{
	         if(backPack == null) return;
	         
	         //Probably want to put a validations check here to make sure backpack is in range
	         //though this is only gonna hurt this player so isnt really hackable lol
	         
	        //Get backpack slots then:
	        
		foreach (var slot in slots)
		{
			if(slot.IsEmpty) continue;
			if (slot.ItemObject.TryGetComponent<MouseTrap>(out var trap))
			{
				if(trap.IsArmed) trap.TriggerTrapFromContainer(playerScript.health);
			}
		}
	}
```

Reply:
This doesn't work idk how can you access slots if you don't even supply it.

Reply:
done